xbmc/network/upnp/UPnPInternal.cpp

```diff
@@ -606,6 +610,82 @@ BuildObject(CFileItem&                    item,
         }
     }
 
+    //Add external subtitle
+    if (upnp_server && item.IsVideo()) //only if video file
+    {
+        // find any available external subtitles
+        std::vector<std::string> filenames;
+        std::vector<std::string> subtitles;
+        CUtil::ScanForExternalSubtitles(file_path.GetChars(), filenames);
+
+        NPT_String ext;
```

Comment:
Please use `std::string` and `StringUtils`. `NPT_String` should only be used where it comes out of the Platinum SDK.